CXL/ruleset.xml

```diff
@@ -171,4 +181,41 @@
         </properties>
     </rule>
 
+    <!-- Force phpDocs separation between different annotations and remove extra `*` -->
+    <!-- Require specific order of phpDoc annotations with empty newline between specific groups -->
+    <rule ref="SlevomatCodingStandard.Commenting.DocCommentSpacing">
+        <properties>
+            <property name="linesCountBetweenAnnotationsGroups" value="1"/>
+            <property name="linesCountBetweenDifferentAnnotationsTypes" value="1"/>
+            <property name="annotationsGroups" type="array">
+                <element value="
+				@ORM\,
+			"/>
+                <element value="
+                    @since,
+                    @see,
+                    @link,
+					@todo,
+            "/>
+                <element value="
+                    @global,
+                    @var,
+            "/>
+                <element value="
+					@var,
+					@param,
+					@return,
+				"/>
+                <element value="
+					@codeCoverageIgnore,
+                    @phpcsSuppress,
+					@dataProvider,
```

Comment:
Add `@psalm-suppress`. Can we use wildcards as well eg. `@psalm-*`?

Reply:
> Add `@psalm-suppress`. Can we use wildcards as well eg. `@psalm-*`?

I am not aware of `*` wildcard use in phpcs rules. We can add other `annotations` later as per needs.

Reply:
Add `@psalm-suppress` then

Reply:
found this: `@SuppressWarnings(PHPMD.ExitExpression)`

Ref: https://github.com/conversionxl/institute-plugin/blob/a29c37b6e8854860522d107445b81b5ca2d0b475/src/Plugin.php#L262

Add it to which group?

Reply:
`@package`

What about this? Usually found at top of files?

Ref: https://github.com/conversionxl/institute-plugin/blob/a29c37b6e8854860522d107445b81b5ca2d0b475/src/Plugin.php#L5

Reply:
I looked at our codebase and also on WooCommerce. I don't see 100% consistency. It's usually used together with `@since`. I think it can go first as you suggested on chat.